src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -1455,4822 +940,1464 @@ public void Dispose(bool disposing)
-                    ThrowTerminatingError(new ErrorRecord(ex, "InvalidParameterForDCOM", ErrorCategory.InvalidOperation, null));
-                }
-
-                if (_isProtocolSpecified && usingWsman && (_isDcomAuthenticationSpecified || _isImpersonationSpecified))
-                {
-                    string errorMsg = StringUtil.Format(ComputerResources.InvalidParameterForWSMan,
-                                                        ComputerResources.ParameterUsage);
-                    InvalidOperationException ex = new InvalidOperationException(errorMsg);
-                    ThrowTerminatingError(new ErrorRecord(ex, "InvalidParameterForWSMan", ErrorCategory.InvalidOperation, null));
-                }
-
-                if (!_isProtocolSpecified && WsmanAuthentication != null)
-                {
-                    // Change the protocol to be WSMan if the WsmanAuthentication is specified
-                    Protocol = ComputerWMIHelper.WsmanProtocol;
-                }
-            }
-
-#if CORECLR
-            if (this.MyInvocation.BoundParameters.ContainsKey("DcomAuthentication"))
-            {
-                string errMsg = StringUtil.Format(ComputerResources.InvalidParameterForCoreClr, "DcomAuthentication");
-                PSArgumentException ex = new PSArgumentException(errMsg, ComputerResources.InvalidParameterForCoreClr);
-                ThrowTerminatingError(new ErrorRecord(ex, "InvalidParameterForCoreClr", ErrorCategory.InvalidArgument, null));
-            }
-
-            if (this.MyInvocation.BoundParameters.ContainsKey("Impersonation"))
-            {
-                string errMsg = StringUtil.Format(ComputerResources.InvalidParameterForCoreClr, "Impersonation");
-                PSArgumentException ex = new PSArgumentException(errMsg, ComputerResources.InvalidParameterForCoreClr);
-                ThrowTerminatingError(new ErrorRecord(ex, "InvalidParameterForCoreClr", ErrorCategory.InvalidArgument, null));
-            }
-
-            // DCOM Authentication is not supported for CoreCLR. Throw an error
-            // and request that the user specify WSMan Authentication.
-            if (_isDcomAuthenticationSpecified ||
-                Protocol.Equals(ComputerWMIHelper.DcomProtocol, StringComparison.OrdinalIgnoreCase))
-            {
-                InvalidOperationException ex = new InvalidOperationException(ComputerResources.InvalidParameterDCOMNotSupported);
-                ThrowTerminatingError(new ErrorRecord(ex, "InvalidParameterDCOMNotSupported", ErrorCategory.InvalidOperation, null));
-            }
-
-            // TODO:CORECLR This should be re-visited if we decide to add double hop remoting to CoreCLR (outgoing connections)
-            if (ParameterSetName.Equals(AsJobParameterSet, StringComparison.OrdinalIgnoreCase))
-            {
-                InvalidOperationException ex = new InvalidOperationException(ComputerResources.InvalidParame
```

Comment:
remove DCOM.  Perhaps we should just remove Protocol as a paramter until we support something other than WSMan (applies to other cmdlets as well)